fix permission problems with vobcopy, see

http://www.vdr-portal.de/board/thread.php?postid=561530#post561530
http://www.vdr-portal.de/board/thread.php?postid=561577#post561577

Joerg Bornkessel <hd_brummy@gentoo.org> 06 Jan 2007
diff -Naur dvdconvert-1.0.2.orig/scripts/dvd2dvd-1.0.2.sh dvdconvert-1.0.2/scripts/dvd2dvd-1.0.2.sh
--- dvdconvert-1.0.2.orig/scripts/dvd2dvd-1.0.2.sh	2007-01-06 21:06:21.535061777 +0100
+++ dvdconvert-1.0.2/scripts/dvd2dvd-1.0.2.sh	2007-01-06 21:12:44.440882336 +0100
@@ -708,20 +708,22 @@
 		    sleep $SLEEPTIME
 		fi
 	done
+	
+	[[ -e ${WORKDIR}/vobcopy_*.log ]] && rm -f ${WORKDIR}/vobcopy_*.log
 
 	case $ACTION_GETTITLE in
 	    ${GETTITLE_TOOLS[0]})
 		;;
 	    ${GETTITLE_TOOLS[1]})
 		d2v_log "`echo && echo \$\> $VOBCOPY_BIN -i $DVD_DEVICE \
-		    -v -v -I -L ${TEMPDIR} 2\> /dev/null`"
+		    -v -v -I -L ${WORKDIR} 2\> /dev/null`"
 		$VOBCOPY_BIN -i $DVD_DEVICE -v -v -I \
-		    -L ${TEMPDIR} 2> /dev/null && \
-		TITLE=`cat ${TEMPDIR}/vobcopy_*.log | \
+		    -L ${WORKDIR} 2> /dev/null && \
+		TITLE=`cat ${WORKDIR}/vobcopy_*.log | \
 		    grep 'Name of the dvd' | head -n 1 | awk '{print $6}'` && \
-		cat ${TEMPDIR}/vobcopy_*.log >> $LOGFILE
+		cat ${WORKDIR}/vobcopy_*.log >> $LOGFILE
 		if [ $TITLENUM = "AUTO" ] ; then
-			TITLENUM=`cat ${TEMPDIR}/vobcopy_*.log | \
+			TITLENUM=`cat ${WORKDIR}/vobcopy_*.log | \
 			grep 'Using Title' | head -n 1 | awk '{print $4}'`
 		fi
 		;;
@@ -1549,6 +1551,7 @@
 
 [ $ACTION = "all" -o $CLEAN = 1 ] && rm -rf $TEMPDIR && d2v_log "\$TEMPDIR '$TEMPDIR' deleted"
 
+rm -f ${WORKDIR}/vobcopy_*.log
 rm -f $LOCKFILE
 
 d2v_log_separator
diff -Naur dvdconvert-1.0.2.orig/scripts/dvd2vdr-1.0.1.sh dvdconvert-1.0.2/scripts/dvd2vdr-1.0.1.sh
--- dvdconvert-1.0.2.orig/scripts/dvd2vdr-1.0.1.sh	2007-01-06 21:06:21.531061547 +0100
+++ dvdconvert-1.0.2/scripts/dvd2vdr-1.0.1.sh	2007-01-06 21:14:29.690880190 +0100
@@ -640,20 +640,22 @@
 		    sleep $SLEEPTIME
 		fi
 	done
+	
+    [[ -e ${WORKDIR}/vobcopy_*.log ]] && rm -f ${WORKDIR}/vobcopy_*.log
 
 	case $ACTION_GETTITLE in
 	    ${GETTITLE_TOOLS[0]})
 		;;
 	    ${GETTITLE_TOOLS[1]})
 		d2v_log "`echo && echo \$\> $VOBCOPY_BIN -i $DVD_DEVICE \
-		    -v -v -I -L ${TEMPDIR} 2\> /dev/null`"
+		    -v -v -I -L ${WORKDIR} 2\> /dev/null`"
 		$VOBCOPY_BIN -i $DVD_DEVICE -v -v -I \
-		    -L ${TEMPDIR} 2> /dev/null && \
+		    -L ${WORKDIR} 2> /dev/null && \
 		TITLE=`cat ${TEMPDIR}/vobcopy_*.log | \
 		    grep 'Name of the dvd' | head -n 1 | awk '{print $6}'` && \
-		cat ${TEMPDIR}/vobcopy_*.log >> $LOGFILE
+		cat ${WORKDIR}/vobcopy_*.log >> $LOGFILE
 		if [ $TITLENUM = "AUTO" ] ; then	
-			TITLENUM=`cat ${TEMPDIR}/vobcopy_*.log | \
+			TITLENUM=`cat ${WORKDIR}/vobcopy_*.log | \
 			grep 'Using Title' | head -n 1 | awk '{print $4}'`
 		fi
 		;;
@@ -1371,6 +1373,7 @@
 
 [ $ACTION = "all" -a $CLEAN = 1 ] && rm -rf $TEMPDIR && d2v_log "\$TEMPDIR '$TEMPDIR' deleted"
 
+rm -f ${WORKDIR}/vobcopy_*.log
 rm -f $LOCKFILE
 
 d2v_log_separator

